.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.6.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk-15.0.2.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/nlutale/Library/Application Support/JetBrains/IntelliJIdea2021.3/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/nlutale/Library/Application%20Support/JetBrains/IntelliJIdea2021.3/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
This file shouldn't be merged. Should probably be added to the `.gitignore` file

Reply:
🤦‍♂️ , I did a `git add .`